doc/sphinx-guides/source/user/dataset-management.rst

```diff
@@ -180,11 +180,13 @@ Metadata found in the header section of `Flexible Image Transport System (FITS)
 Compressed Files
 ----------------
 
-Compressed files in zip format are unpacked automatically. If it fails to unpack, for whatever reason, it will upload as is. If the number of files inside are more than a set limit (1,000 by default, configurable by the Administrator), you will get an error message and the zip file will uploads as is.
+Compressed files in .zip format are unpacked automatically. If a .zip file fails to unpack for whatever reason, it will upload as is. If the number of files inside are more than a set limit (1,000 by default, configurable by the Administrator), you will get an error message and the .zip file will upload as is.
+
+If the uploaded .zip file contains a folder structure, Dataverse will keep track of this structure. A file's location within this folder structure is displayed in the file metadata as the File Path. When you download the contents of the dataset, this folder structure will be preserved and files will appear in their original locations. 
```

Comment:
Do I get the exact same folder structure as I originally uploaded? Here's an example of how we changed directory names and sometimes file names in the DVN 3 days: https://github.com/IQSS/dataverse-client-r/issues/18#issuecomment-327268497

## The zip file I uploaded to DVN 3

```
murphy:zip pdurbin$ zip -r data.zip folder1 folder2 -x "*.DS_Store"
  adding: folder1/ (stored 0%)
  adding: folder1/sub1/ (stored 0%)
  adding: folder1/sub1/file.txt (stored 0%)
  adding: folder1/sub2/ (stored 0%)
  adding: folder1/sub2/file.txt (stored 0%)
  adding: folder2/ (stored 0%)
  adding: folder2/sub1/ (stored 0%)
  adding: folder2/sub1/file.txt (stored 0%)
  adding: folder2/sub2/ (stored 0%)
  adding: folder2/sub2/file.txt (stored 0%)
murphy:zip pdurbin$ 
```

## The zip file I downloaded from DVN 3

```
murphy:unzip pdurbin$ unzip study_10000.zip 
Archive:  study_10000.zip
  inflating: folder1-sub1/file.txt   
  inflating: folder1-sub2/file_1.txt  
  inflating: folder2-sub1/file_2.txt  
  inflating: folder2-sub2/file_3.txt  
  inflating: MANIFEST.TXT            
murphy:unzip pdurbin$ 
```

Reply:
I just tested this on demo (thanks for the suggestion at standup) and here's how it works as of Dataverse 4.11.

## The zip file I uploaded to Dataverse 4.11

```
murphy:uploaded pdurbin$ unzip data.zip 
Archive:  data.zip
   creating: folder1/
   creating: folder1/sub1/
 extracting: folder1/sub1/file.txt   
   creating: folder1/sub2/
 extracting: folder1/sub2/file.txt   
   creating: folder2/
   creating: folder2/sub1/
 extracting: folder2/sub1/file.txt   
   creating: folder2/sub2/
 extracting: folder2/sub2/file.txt   
murphy:uploaded pdurbin$ 
```

## The zip file I downloaded from Dataverse 4.11

```
murphy:downloaded pdurbin$ unzip dataverse_files.zip 
Archive:  dataverse_files.zip
   creating: folder1/sub1/
  inflating: folder1/sub1/file.txt   
   creating: folder1/sub2/
  inflating: folder1/sub2/file.txt   
   creating: folder2/sub1/
  inflating: folder2/sub1/file.txt   
   creating: folder2/sub2/
  inflating: folder2/sub2/file.txt   
  inflating: MANIFEST.TXT            
murphy:downloaded pdurbin$ 
```

This is looking good to me. Dataverse adds the MANIFEST.TXT file, but this is a feature. Here is the content of that file:

```
murphy:downloaded pdurbin$ cat MANIFEST.TXT 
folder1/sub1/file.txt (text/plain) 16 bytes.
folder1/sub2/file.txt (text/plain) 16 bytes.
folder2/sub1/file.txt (text/plain) 13 bytes.
folder2/sub2/file.txt (text/plain) 13 bytes.
murphy:downloaded pdurbin$ 
```